api/src/db.rs

```diff
@@ -319,7 +316,7 @@ impl Database {
         Ok(records.into_iter().map(|r| r.weapon_skin_id).collect())
     }
 
-    pub(crate) async fn get_sales_without_bullshit(&self, skin_id: i32) -> Result<Vec<Sale>> {
```

Comment:
lol rust rover saw me drag it into a library crate and made them public

Reply:
Wow